Added embeeded tweets, fixed padding, added Testimonials

MikeStrider opened the pull request:
Added embeeded tweets, fixed padding, added Testimonials

BennyCarlsson commented:
Looks great however there seems to be some merge conflicts probably because I merged some pull requests before this one. If you have time to fix it or make a new pull requests I'll happily merge it :)

MikeStrider commented:
ya. should be able to tonight.


regards'
M Strong




-------- Original message --------
From: Benny Carlsson <notifications@github.com>
Date: 2019-10-21 5:31 a.m. (GMT-05:00)
To: BennyCarlsson/MyPortfolio-Hacktoberfest2019 <MyPortfolio-Hacktoberfest2019@noreply.github.com>
Cc: Michael Strong <mike.strong@live.ca>, Author <author@noreply.github.com>
Subject: Re: [BennyCarlsson/MyPortfolio-Hacktoberfest2019] Added embeeded tweets, fixed padding, added Testimonials (#263)


Looks great however there seems to be some merge conflicts probably because I merged some pull requests before this one. If you have time to fix it or make a new pull requests I'll happily merge it :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/BennyCarlsson/MyPortfolio-Hacktoberfest2019/pull/263?email_source=notifications&email_token=AIFKGDBWOVPAIJLQLATZYXTQPVZFPA5CNFSM4JCXOXLKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEBZV4NI#issuecomment-544431669>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AIFKGDF5SPLRK4JWGDSINRDQPVZFPANCNFSM4JCXOXLA>.

BennyCarlsson commented:
Great 👍
